js/components/portal-dashboard/question-level-feedback-student-rows.tsx

```diff
@@ -48,7 +51,7 @@ export const QuestionLevelFeedbackStudentRows: React.FC<IProps> = (props) => {
     }
 
     return (
-      <div key={index} className={css.feedbackRows__row}>
+      <div key={currentQuestionId + index.toString()} className={css.feedbackRows__row}>
```

Comment:
Same as above -- as long as the `currentQuestionId` is unique, you shouldn't add the `index`.

Reply:
Ah, the importance of understanding the context rather than just looking at a line of code in isolation. In that case, yes, you do need more that the `activityId` or `currentQuestionId`. An ID associated with the student would be better than an index, however, if one is available, for the reasons mentioned above.